Error while building on Windows 11 ("misc\odin.rc(41) : error RC2104 : undefined keyword or key name: All")

My System: Windows 11x64

I've downloaded the master version (and also the dev-2024-11 version) and I can't build Odin because the following error occurs:

```bat
misc\odin.rc(41) : error RC2104 : undefined keyword or key name: All
main.cpp
libtommath.cpp
LINK : fatal error LNK1181: cannot open input file 'misc\odin.res'

mt : general error c101008c: Failed to read the manifest from the resource of file "\\?\R:\Sources\Odin\Odin\odin.exe".
```

I've made some tests, to find the cause of the issue and I found that the `build.bat file` of the `dev-2024-10` works well and if I replace the master version by this one, I can build Odin.

The changes in the file are very small. The line that causes the error in the build is :

In the master version (build is HS):
```bat
for /f %%i in ('powershell get-date -format "{yyyyMMdd}"') do (
    set CURR_DATE_TIME=%%i
)
```

In the dev-2024-10 version (build is OK):
```bat
for /f "usebackq tokens=1,2 delims=,=- " %%i in (`wmic os get LocalDateTime /value`) do @if %%i==LocalDateTime (
    set CURR_DATE_TIME=%%j
)
```
(note that this line of script is buggy, because the loop uses %%i and the CURR_DATE_TIME uses %%j,I've tested to put `%%i` instead of `%%j` and nothing changes

To debug the code deeper, I've added the line:
```bat
echo %odin_version_full%
```
after the line:
```bat
set odin_version_full="%V1%.%V2%.%V3%.%V4%"
```

in both files (master and dev-2024-10) and these are the results:

with the dev-2024-10 version -> "2024.11.0.0" is printed
with the master version -> "Loca.lD.0.0" is printed

I ran the command from cmd.exe.
```bat
powershell get-date -format "{yyyyMMdd}" 
```

I've got the error message:

```bat
The argument 'get-date' is not recognized as the name of a script file. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
same error in a powershell terminal or in a command prompt console (cmd.exe)
```

I have this error when building on my windows machine, looks slightly different from yours though:
```
> build.bat
libtommath.cpp
main.cpp
LINK : fatal error LNK1104: cannot open file 'odin.exe'

mt : manifest authoring error c1010001: Values of attribute "version" not equal in different manifest snippets.
The system cannot find the batch label specified - end_of_build
```

> > build.bat
> libtommath.cpp
> main.cpp
> LINK : fatal error LNK1104: cannot open file 'odin.exe'
> 
> mt : manifest authoring error c1010001: Values of attribute "version" not equal in different manifest snippets.
> The system cannot find the batch label specified - end_of_build

Perhaps could you test 

> ```batchfile
> powershell get-date -format "{yyyyMMdd}" 
> ```

in your powershell prompt to see if the command still exists for you ?

--- a/build.bat
+++ b/build.bat
@@ -19,7 +19,11 @@ if "%VSCMD_ARG_TGT_ARCH%" neq "x64" (
 	)
 )
 
-for /f %%i in ('powershell get-date -format "{yyyyMMdd}"') do (
+pushd misc
+cl get-date.c
+popd
+
+for /f %%i in ('misc\get-date') do (
 	set CURR_DATE_TIME=%%i
 )
 set curr_year=%CURR_DATE_TIME:~0,4%
@@ -57,7 +61,6 @@ if %release_mode% equ 0 (
 set V4=0
 set odin_version_full="%V1%.%V2%.%V3%.%V4%"
 set odin_version_raw="dev-%V1%-%V2%"
-
 
 set compiler_flags= -nologo -Oi -TP -fp:precise -Gm- -MP -FC -EHsc- -GR- -GF
 rem Parse source code as utf-8 even on shift-jis and other codepages

--- a/misc/get-date.c
+++ b/misc/get-date.c
@@ -1,0 +1,13 @@
+/*
+	Prints the current date as YYYYMMDD
+
+	e.g. 2024-12-25
+*/
+#include <stdio.h>
+#include <time.h>
+
+int main(int arg_count, char const **arg_ptr) {
+	time_t t = time(NULL);
+	struct tm* now = localtime(&t);
+	printf("%04d%02d%02d", now->tm_year + 1900, now->tm_mon + 1, now->tm_mday);
+}

